R/quiz.R

```diff
@@ -1,81 +1,182 @@
 
-# Make sure choose answers is > than the number of answers you give. lol
-# Make sure that if you say "choose-answers" you use the C) m) o) notation
-# Make sure if you don't use choose answers but instead use : a, b,c,d you delete the "choose-answers" tag
-# Don't have exclamation points in answers.
-# Make sure there's at least one right answer!
-# Make sure the top of the quiz tag looks something like: {quiz, id: quiz_why_doc, attempts: 10}
-# Make sure all quizzes are listed in Book.txt (I think John made a check for this part already).
+# These will be existing paths to grab from for examples
+#' @export
+good_quiz_path <- list.files(pattern = "quiz_good.md",
```

Comment:
I suspect this is part of the problem because its' trying to run this before `leanbuild` is actually installed.